yarn.lock

```diff
@@ -13194,7 +13193,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"ember-destroyable-polyfill@npm:^2.0.1, ember-destroyable-polyfill@npm:^2.0.2, ember-destroyable-polyfill@npm:^2.0.3":
+"ember-destroyable-polyfill@npm:^2.0.1, ember-destroyable-polyfill@npm:^2.0.2":
```

Comment:
[thought] @alex-ju @shleewhite would it be worth to run a `yarn dedupe` in general (in a different PR, maybe; or maybe not)? as far as I remember, @alex-ju did this operation with some cadence

Reply:
yes, we do `yarn dedupe` when upgrading packages. we're due an update soon